src/CMakeLists.txt

```diff
@@ -327,12 +327,6 @@ IF(NOT USE_BUNDLED_ZLIB)
 	IF(ZLIB_FOUND)
 		LIST(APPEND LIBGIT2_SYSTEM_INCLUDES ${ZLIB_INCLUDE_DIRS})
 		LIST(APPEND LIBGIT2_LIBS ${ZLIB_LIBRARIES})
-		IF(APPLE OR CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
-			LIST(APPEND LIBGIT2_LIBS "z")
-			LIST(APPEND LIBGIT2_PC_LIBS "-lz")
```

Comment:
I agree that the first line is definitely bogus, we don't want to append twice to `LIBGIT2_LIBS`. But appending to `LIBGIT2_PC_LIBS` is in fact required. Doing so will not cause us to link twice, but the `PC_LIBS` variable will cause us to append this lib to the generated pkgconfig file.

Reply:
Agree -- updates committed.